bin/minifollowups/pycbc_page_coincinfo

```diff
@@ -43,6 +43,10 @@ trig_input.add_argument('--n-loudest', type=int,
 trig_input.add_argument('--trigger-id', type=int,
     help="Examine the trigger with specified ID, use with statmap file. An "
          "alternative to --n-loudest. Cannot be used together")
+parser.add_argument('--include-summary-page-link', action='store_true',
```

Comment:
is there a specific reason to make this optional?

Reply:
Yes. I think I don't want to include this link by default (and it will certainly not be useful for outside folks, who cannot access these pages anyway). However, there might still be a use-case for this, so I allow this to be added into the table with this flag if required.

Reply:
well, should make sure to remember to put this option into LSC production ini files to maintain functionality.